cli/lib/upgrade.go

```diff
@@ -39,7 +37,7 @@ func initUpgradeCommand() *cobra.Command {
 		Long: "This command ungrades all installed libraries to the latest available version." +
 			"To upgrade a single library use the 'install' command.",
 		Example: "  " + os.Args[0] + " lib upgrade",
```

Comment:
The example could be enriched showing something like 
```
Examples:
 ./arduino-cli lib upgrade
./arduino-cli lib upgrade Audio
 ./arduino-cli lib upgrade Audio ArduinoJson
```
in order to show clearly that the update for multiple libraries at once is supported